test/InputOutput/hybrid3dbox.jl

```diff
@@ -0,0 +1,124 @@
+using Test
+using StaticArrays, IntervalSets, LinearAlgebra, UnPack
+
+import ClimaCore:
+    ClimaCore,
```

Comment:
```suggestion
import ClimaCore
import ClimaCore:
```
I don't think that this should matter, but I've not really used this syntax before

Reply:
Also, there are a couple of these